src/OTSubscriber.js

```diff
@@ -156,7 +159,9 @@ OTSubscriber.defaultProps = {
   session: null,
   className: '',
   style: {},
-  properties: {},
+  properties: {
+    insertDefaultUI: true,
```

Comment:
No need to set `insertDefaultUI: true`.
The insertDefaultUI default value is true, so if you don't define the value, it will fallback to true. Please remove it and leave `properties:{}`.

Reply:
@enricop89 without `insertDefaultUI: true`, there is a test case which is failing, so do you recommend to change that test case also?

Reply:
Sorry for the delay. I tested again your PR, and you're right. Let's leave the default value to true.
Thanks again!